sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -171,6 +171,10 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+## Developer Contributing Guide
+
+For details on developer inner-loop workflow and contributing guide see the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md)
```

Comment:
Why does this section need a separate header?
Even if it does it should not be at the same level as "Contributing"
I vote for no header. The existing section is already about "Contributing", so that should be enough

Reply:
Thanks! This is a discussion I wanted to have because in the issue Jeremy mentioned to add a separate developer.md under each readme. So wanted to confirm what we should do. I can remove the header "Developer Contributing Guide"